### ISSUE TITLE
Create Server-Side Pool Candidate Report Document Generator

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#9686]
## ✨ Feature

We want to start generating the Pool Candidate report documents on the server.  Let's set up the view to do that.

## 🕵️ Details

This issue only deals with the report document itself.  How the client requests it and how it gets back to the client are out of scope for now.

## 🎨 Design File

The server-side report should match the current client-side report design.

## 🙋‍♀️ Proposed Implementation

[Laravel-pdf](https://spatie.be/docs/laravel-pdf/v1/introduction)
[PHPOffice    /    PHPWord](https://github.com/PHPOffice/PHPWord)

## ✅ Acceptance Criteria

- [ ] It is possible to generate a document with one or more pool candidates on the server
  - [ ] A dev can trigger the request manually.  A regular user cannot.
  - [ ] No need to authenticate request yet
  - [ ] No need to save the file on the server, can just send back an HTTP response
- [ ] Report matches the report generated by the client

## 🛑 Blockers

Issues which must be completed before this one.

```[tasklist]
### Blocked By
```

**Comment by tristan-orourke** [comment]
@gobyrne and I decided - Word doc is better than PDF, for our users and for accessibility. If producing the doc as a Word doc isn't overly difficult, let's do that.

**Comment by petertgiles** [comment]
> I remember someone mentioning they would prefer separate documents per candidate and zip those files. Is this still the goal?
@Jerryescandon

**Comment by Jerryescandon** [comment]
> I remember someone mentioning they would prefer separate documents per candidate and zip those files. Is this still the goal?

That is correct

**Comment by tristan-orourke** [comment]
That is definitely the goal. I've been thinking that would be worth its own ticket, but if that influences the planning on this one, yes, please keep it in mind.
